scripts/zones/Norg/npcs/Stray_Cloud.lua

```diff
@@ -1,18 +1,58 @@
 -----------------------------------
 -- Area: Norg
 --  NPC: Stray Cloud
--- Standard Info NPC
+-- Starts and Ends Quest: An Undying Pledge
+-- !pos-20.617, 1.097, -29.165, 133
 -----------------------------------
+require("scripts/globals/titles")
```

Comment:
titles, settings, and NorgIDs requires are not used in this script, so aren't needed